bindata/manifests/metrics-exporter/metrics-config-map.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriov-network-metrics-exporter-config
+  namespace: {{.Namespace}}
+data:
+  drivers.yaml: |-
+    drivers:
```

Comment:
How these settings work? Can we rely on these hardcoded values for all environments?